pkg/Library/Goog/IsPoisonous.js

```diff
@@ -41,19 +41,63 @@ isPoisonous(response) {
 
 async runMacro(macroId, inputs, service) {
   const data = {macroId, inputs};
-  return await service({kind: 'GoogleApisService', msg: 'runMacro', data});
+  return service({kind: 'GoogleApisService', msg: 'runMacro', data});
 },
 
 render({entityName}, {isPoisonous}) {
+  const q = `Is "${entityName}" poisonous?`;
+  const a = isPoisonous === null
+    ? '[considering...]'
+    : isPoisonous
+    ? `may indeed be poisonous (or venomous), be advised.`
+    : `might be safe, but I can make mistakes!`
```

Comment:
haha, i like the wording :)
reminds me of a joke:
- what are the chances to meet a dinosaur on the street?
- about 50% - you either meet it, or not.

Reply:
Hahaha. I can sell you a Dinosaur-Keep-Away bracelet for only $250.